src/main/java/com/dougnoel/sentinel/steps/DownloadVerificationSteps.java

```diff
@@ -65,4 +70,28 @@ public static void verifyFileWithFilenameDownloaded(String expectedFilename) thr
 	public static void clearDownloadsFolder() throws IOException {
 		DownloadManager.clearDownloadDirectory();
 	}
+
+	/**
+	 * Looks inside the most recently downloaded zip file and asserts that it does or does not contain a file with the given extension in the top-level directory.
+	 * @param assertion String " does not" for a negative check, otherwise positive check
+	 * @param expectedFileType String the file extension to check for.
+	 */
+	@Then("^I verify the most recently downloaded zip file( does not)? contains? a file with the extension (.*?)$")
```

Comment:
Nothing wrong with the function, but it seems like an odd case to include. My instinct would be to have a function to check if a specific file is present inside a zip, not any extension type.

Reply:
Like you could check for anything with an extension with 
"I verify the most recently downloaded zip file does not contains a file with the name *.exe"

Reply:
Fair but that's not my use case. I don't care what the name is, and implementing regex / wildcard behavior like that seems like overkill. My opinion is if we need it later we can add it.

Reply:
I'm ok with that. Filename is a different use case we can implement when needed.